omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html

```diff
@@ -99,14 +99,9 @@
                 <div th:each="note: ${resource.note}" th:text="${note.text}"></div>
             </td>
         </tr>
-        </tbody>
-    </table>
-    <th:block th:each="reaction: ${resource.reaction}">
-        <br/>
-        <table class="hapiPropertyTable">
-            <tbody>
+        <th:block th:each="reaction: ${resource.reaction}">
             <tr>
-                <td>Reaction:</td>
+                <td><br/>Reaction:</td>
```

Comment:
Is this intentional? It looks weird to me having a line break inside a `<td>`.

Reply:
Actually it was looking a bit cluttered without any separation, so I added a newline. Should I remove it then?